### ISSUE TITLE
Command-line tool needs extension?

### ISSUE THREAD

**Issue by treatmesubj** [google/importlab#51]
```
pip install importlab
```
Expected to use the importlab command-line tool on Windows but:
```
'importlab' is not recognized as an internal or external command,
operable program or batch file.
```
I saw importlib was in fact sitting in  C:\Users\...\AppData\Local\Programs\Python\Python38-32\Scripts, but I guess it wouldn't do anything without an extension, so I just renamed it to runimportlab.py to avoid circular reference and works fine. 

```
runimportlab.py --tree script.py 
```

**Comment by rchen152** [comment]
I don't think we test on Windows, so no surprise importlab doesn't work. If you're using Windows 10, you might be able to get around this using WSL. The "Requirements" and "Installing" sections of the README for pytype (a project that uses importlab for which users have contributed some bare-bones WSL installation instructions) might be helpful: https://github.com/google/pytype#requirements.
